service-print/pom.xml

```diff
@@ -37,6 +37,7 @@
         <dependency>
             <groupId>org.apache.pdfbox</groupId>
             <artifactId>pdfbox</artifactId>
+            <version>${pdfbox.version}</version>
```

Comment:
Version should be removed since we have pdfbox in parent-pom dependencyManagement.
```suggestion
```